joomla/joomla-cms
# [4.0] Split the rules field to 2 layouts

### Summary of Changes
Splits the HTML part (JLayout) into two files. 
Reasons:
1: less code per file
2: more efficient caching (less allocate memory, etc)


### Testing Instructions
1: Apply patch
2: Go to Admin->Global Configuration
3: Check Permissions working as before

### Expected result
This is code improvement

### Actual result
Same as before


### Documentation Changes Required
Yes
